tests/llm/test_vertexai/test_modes.py

```diff
@@ -92,3 +92,30 @@ def test_complex_nested_model(model, mode):
     assert {book.author for book in resp.books} == {"Jane Doe", "John Smith"}
     assert {book.genre for book in resp.books} == {"Fantasy", "Non-Fiction"}
     assert {book.isbn for book in resp.books} == {"1234567890", "0987654321"}
+
```

Comment:
Let's add one more test for image understanding and should be good to merge. Going to just approve it first

Reply:
Maybe something like this just to make sure we don't throw any errors in the `test_vertexai` folder

```
client = instructor.from_vertexai(
    gm.GenerativeModel("gemini-1.5-pro-001"), mode=instructor.Mode.VERTEXAI_JSON
)
content = [
    gm.Part.from_text("What country is this in?"),
    gm.Part.from_uri(
        "gs://cloud-samples-data/vertex-ai/llm/prompts/landmark2.png",
        mime_type="image/png",
    ),
]


class Description(BaseModel):
    description: str


resp = client.create(
    response_model=Description,
    messages=content,
)

print(resp)
```